package.json

```diff
@@ -9,13 +9,17 @@
   "license": "MIT",
   "author": "Exodus Movement, Inc.",
   "main": "./lib/index.js",
+  "files": [
+    "src/"
+  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ExodusMovement/buffer-noise.git"
   },
   "scripts": {
-    "build": "flow-remove-types ./src/index.js > ./lib/index.js",
+    "build": "make-dir lib && flow-remove-types ./src/index.js > ./lib/index.js",
```

Comment:
If we are not shipping transpiling code, why are we building? Should lint be enough to detect any issue babel can detect? Can we remove babel?

There are general questions, no just for this PR